swcregistrypython/get_entry_info.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import json
+
+
+class SWC:
+    def __init__(self, swc_id):
+        self.swc_id = swc_id
+        self.path_file_content = 'swc-definition.json'
+
+    def get_content_by_file(self):
```

Comment:
Shouldn't this be private?
```suggestion
    def _get_content_by_file(self):
```